editor/project_manager.cpp

```diff
@@ -499,6 +499,7 @@ class ProjectDialog : public ConfirmationDialog {
 					initial_settings["application/config/name"] = project_name->get_text();
 					initial_settings["application/config/icon"] = "res://icon.png";
 					initial_settings["rendering/environment/default_environment"] = "res://default_env.tres";
+					initial_settings["physics/common/enable_pause_aware_picking"] = true;
```

Comment:
Shouldn't it be disabled on new projects if we decide to default to the old behavior on 3.2? Otherwise people might have compatibility breakage when they use an existing script on a different project.

Reply:
My idea is to make it the new default so new projects benefit from the new behavior with zero setup.
However, you have a point in the cross-project compatibility rationale.
I don't have a strong opinion, but I think the benefit-to-risk ratio is higher with the new default.

Reply:
Given that the old behavior is mostly buggy, on second thought I would agree it's best to introduce the new behavior by default on new projects.